test/docker-e2e/e2e_simple_test.go

```diff
@@ -72,4 +80,40 @@ func (s *CelestiaTestSuite) TestE2ESimple() {
 			t.Fatalf("Timed out waiting for %d transactions", requiredTxs)
 		}
 	}
+
+	testBankSend(t, celestia, cfg)
+}
+
+// testBankSend performs a basic bank send using txClient.
+func testBankSend(t *testing.T, chain *tastoradockertypes.Chain, cfg *dockerchain.Config) {
```

Comment:
this is kind of just a PoC fn to verify the txClient can broadcast transactions properly. It can be removed in the future as more robust mechanisms are put in place in E2Es.